pkg/labels/zz_generated.deepcopy.go

```diff
@@ -1,82 +0,0 @@
-// +build !ignore_autogenerated
```

Comment:
I might be wrong but I think we need a deepcopy function for kubernetes, try doing `make generate-k8s-api`

Reply:
We don't have references to any of the generated functions anywhere in the repo anymore, so I don't see how they would be needed? I did not find how this file was gnerated, however. Wuld this be re-generated by `make generate-k8s-api`?

Reply:
@jrajahalme remove the `pkg/labels/zz_generated.deepcopy.go` and apply the following diff.
```
diff --git a/pkg/labels/doc.go b/pkg/labels/doc.go
index db47aa6b8..7931b1922 100644
--- a/pkg/labels/doc.go
+++ b/pkg/labels/doc.go
@@ -12,9 +12,5 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +k8s:deepcopy-gen=package
-// +k8s:openapi-gen=true
-
 // Package api defines the API of the Cilium network policy interface
-// +groupName=pkg
 package labels
```

Reply:
@jrajahalme missing?